log/klogv2/klogv2.go

```diff
@@ -96,3 +112,22 @@ func (l *adapter) WithFields(fields log.Fields) log.Logger {
 func (l *adapter) WithError(err error) log.Logger {
 	return l.WithField("err", err)
 }
+
+// processFields prepares a string to be appended to every log call.
+// This is called once to avoid costly log operations.
+func processFields(fields map[string]interface{}) string {
```

Comment:
Have you used: https://godoc.org/k8s.io/apimachinery/pkg/util/sets#StringKeySet

Reply:
No and prefer to keep this Go standard string processing.